main.go

```diff
@@ -112,22 +122,23 @@ func parseFlags() *config {
 	}
 
 	a.Flag("googleAPIjsonkeypath", "Path to json keyfile for GCP service account. JSON keyfile also contains project_id").
-		Default("").StringVar(&cfg.googleAPIjsonkeypath)
+		Envar("PROMBQ_GCP_JSON").StringVar(&cfg.googleAPIjsonkeypath)
 	a.Flag("googleAPIdatasetID", "Dataset name as shown in GCP.").
-		Default("").StringVar(&cfg.googleAPIdatasetID)
+		Envar("PROMBQ_DATASET").StringVar(&cfg.googleAPIdatasetID)
 	a.Flag("googleAPItableID", "Table name as showon in GCP.").
-		Default("").StringVar(&cfg.googleAPItableID)
+		Envar("PROMBQ_TABLE").StringVar(&cfg.googleAPItableID)
 	a.Flag("send-timeout", "The timeout to use when sending samples to the remote storage.").
-		Default("30s").DurationVar(&cfg.remoteTimeout)
+		Envar("PROMBQ_TIMEOUT").Default("30s").DurationVar(&cfg.remoteTimeout)
 	a.Flag("web.listen-address", "Address to listen on for web endpoints.").
-		Default(":9201").StringVar(&cfg.listenAddr)
+		Envar("PROMBQ_LISTEN").Default(":9201").StringVar(&cfg.listenAddr)
 	a.Flag("web.telemetry-path", "Address to listen on for web endpoints.").
-		Default("/metrics").StringVar(&cfg.telemetryPath)
+		Envar("PROMBQ_TELEMETRY").Default("/metrics").StringVar(&cfg.telemetryPath)
 
 	flag.AddFlags(a, &cfg.promlogConfig)
 
 	_, err := a.Parse(os.Args[1:])
 	if err != nil {
+		version.Print()
```

Comment:
I don't think it makes sense to print version information if command line argument parsing fails.